src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php

```diff
@@ -134,6 +134,7 @@ protected function loadConsumers(ContainerBuilder $container, array $config)
                     new Reference(sprintf('m6_web_amqp.connection.%s', $consumer['connection'])),
                     $consumer['exchange_options'],
                     $consumer['queue_options'],
+                    $consumer['qos_options'],
```

Comment:
`$consumer['qos_options']` should be after `$lazy` if you look at the definition of `ConsumerFactory::get()`